examples/content-based-routing/content_based_routing.bal

```diff
@@ -1,20 +1,18 @@
-import ballerina/mime;
 import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
 
-@Description {value:"Define the attributes associated with the service endpoint here."}
-endpoint http:Listener cbrEP {
-    port:9090
-};
-
-@Description {value:"Define the attributes associated with the client endpoint here."}
+//Define the attributes associated with the client endpoint here.
```

Comment:
need to add spaces after // ?

Reply:
We discussed it. Done all the changes without spaces. That will not affect the documentation generation using comments